hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ContractMintHTSSuite.java

```diff
@@ -120,16 +126,15 @@ List<HapiSpec> positiveSpecs() {
     @HapiTest
     final HapiSpec transferNftAfterNestedMint() {
         final var nestedTransferTxn = "nestedTransferTxn";
+        final var v2SecuritySendNftAfterNestedMint = "v2SecuritySendNftAfterNestedMint";
 
-        return defaultHapiSpec(
-                        "TransferNftAfterNestedMint",
-                        NONDETERMINISTIC_CONSTRUCTOR_PARAMETERS,
```

Comment:
Is there a reason you removed the fuzzy match params?

Reply:
good catch!